.circleci/config.yml

```diff
@@ -294,15 +294,16 @@ jobs:
           name: Count Generated Project Dependencies
           environment:
           command: |
-            MAX_PACKAGES="3050"
+            MAX_PACKAGES="2250"
             total=$(./scripts/count-dependencies.js /tmp/project)
             echo "export TOTAL_PACKAGES=${total}" >> $BASH_ENV;
+            source $BASH_ENV
```

Comment:
Fix for a regression bug where we forgot to apply/source the new environment variable `TOTAL_PACKAGES`.

Reply:
@adamraya since you have access to Datadog, can you check what happened to the data lately? For `generated_project_total_packages`, whether any value is recorded for it or not. Because it looks like we've been sending blanks.

FYI this is where we sent the data from: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/7bede2aae4787f62061b52b0cc423a78f5539cd3/.circleci/config.yml#L341

Reply:
Good catch with the missing `source` call.

Reply:
It looks like Datadog was still receiving data. But it could be that the data is coming from generated projects not using the latest changes, so it's a good thing to fix.